crates/astria-sequencer/src/app/snapshots/astria_sequencer__app__tests_breaking_changes__app_hash_execute_every_action.snap

```diff
@@ -3,36 +3,36 @@ source: crates/astria-sequencer/src/app/tests_breaking_changes.rs
 expression: app.app_hash.as_bytes()
 ---
 [
+  113,
```

Comment:
why did this snapshot change? these changes aren't breaking right, it's because you added a `bridge_withdrawer_address` to the bridge account state in the test?